README.md

```diff
@@ -1,2 +1,17 @@
-# base-plan-skeleton
-Template for all new Chef Base Plans to simplify creation of repositories.
+# freetype
+
+[![Build Status](https://dev.azure.com/chefcorp-partnerengineering/Chef%20Base%20Plans/_apis/build/status/chef-base-plans.freetype?branchName=master)](https://dev.azure.com/chefcorp-partnerengineering/Chef%20Base%20Plans/_build/latest?definitionId=151&branchName=master)
```

Comment:
Should'nt this be above the "# freetype"?

Reply:
I'd say either way is fine as long as it's in there somewhere to begin with ;)